app/server/src/controllers/indexController.ts

```diff
@@ -16,7 +16,7 @@ export default (config) => {
                 const poppunkRequest = request.body as BeebopRunRequest;
                 const {projectHash, projectId, names, sketches} = poppunkRequest;
                 const {redis} = request.app.locals;
-                await userStore(redis).saveProjectHash(request, projectId, projectHash);
+                await userStore(redis).saveHashAndSampleHasRun(request, projectId, projectHash, names);
```

Comment:
Am I right in thinking this is saving multiple sample names, so this method name maybe a little misleading?
```suggestion
                await userStore(redis).saveHashAndSamplesRun(request, projectId, projectHash, names);
```